tls/s2n_async_pkey.c

```diff
@@ -190,6 +190,7 @@ int s2n_async_pkey_sign_async(struct s2n_connection *conn, s2n_signature_algorit
     sign->on_complete                     = on_complete;
     sign->sig_alg                         = sig_alg;
 
+    GUARD(s2n_hash_new(&sign->digest));
```

Comment:
Should this have been a separate PR?

Reply:
Oh ok, it is part of the new async infrastructure. Only the description of the commit ("previously") made it sound like is was a fix for an older bug.

Reply:
Yeah, it's part of the async changes. Previously was referencing previous commit (: